system/Cache/Handlers/RedisHandler.php

```diff
@@ -102,28 +193,8 @@ public function initialize()
     public function get(string $key)
     {
         $key  = static::validateKey($key, $this->prefix);
-        $data = $this->redis->hMGet($key, ['__ci_type', '__ci_value']);
-
-        if (! isset($data['__ci_type'], $data['__ci_value']) || $data['__ci_value'] === false) {
-            return null;
-        }
-
-        switch ($data['__ci_type']) {
-            case 'array':
-            case 'object':
-                return unserialize($data['__ci_value']);
-
-            case 'boolean':
-            case 'integer':
-            case 'double': // Yes, 'double' is returned and NOT 'float'
-            case 'string':
-            case 'NULL':
-                return settype($data['__ci_value'], $data['__ci_type']) ? $data['__ci_value'] : null;
-
-            case 'resource':
-            default:
-                return null;
-        }
+        $data = $this->redis->get($key);
```

Comment:
The change at line 187 makes all the old type determination/storage unnecessary.